src/rules/require-tothrow-message.js

```diff
@@ -1,41 +1,18 @@
-import { argument, expectCase, getDocsUrl, method } from './util';
+import requireToThrowMessage from './require-to-throw-message';
 
-export default {
+const replacedBy = ['require-to-throw-message'];
+
+const requireTothrowMessage = {
+  ...requireToThrowMessage,
   meta: {
+    ...requireToThrowMessage.meta,
     docs: {
-      url: getDocsUrl(__filename),
-    },
-    messages: {
-      requireRethrow: 'Add an error message to {{ propertyName }}()',
+      ...requireToThrowMessage.meta.docs,
+      replacedBy,
```

Comment:
I don't thing this should be under `docs`, should it? just `meta`

Reply:
I'm not sure. I copied what others did and I found examples of both:

[eslint/rules/no-native-reassign.js#L19](https://github.com/eslint/eslint/blob/v3.8.1/lib/rules/no-native-reassign.js#L14-L37)
[typescript-eslint/rules/prefer-interface.ts#L20](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/src/rules/prefer-interface.ts#L6-L21)

Reply:
In the documentation it's not under `docs`: https://eslint.org/docs/developer-guide/working-with-rules#rule-basics

Reply:
Removed `meta.docs.replacedBy`.